apis/core/v1alpha1/featureflagconfiguration_types.go

```diff
@@ -51,10 +53,29 @@ type FlagDSpec struct {
 
 type FeatureFlagSyncProvider struct {
 	Name string `json:"name"`
+	// +optional
+	// +nullable
+	RemoteSyncConfiguration *RemoteSyncConfiguration `json:"remoteSyncConfiguration"`
```

Comment:
We now support multiple versions of the crd (see the v1alpha2 folder). I suppose these will be breaking changes? If yes we would need to introduce a new version of the crd